spec/json-spec.coffee

```diff
@@ -0,0 +1,13 @@
+describe "JSON grammar", ->
+  grammar = null
+
+  beforeEach ->
+    waitsForPromise ->
+      atom.packages.activatePackage("language-json")
+
+    runs ->
+      grammar = atom.grammars.grammarForScopeName("source.json")
+
+  it "parses the grammar", ->
```

Comment:
I'd rename this to `it "exists in the grammar list"` since it doesn't actually validate parsing.